common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -38,6 +38,12 @@
             this.displayTabTooltip = function(event) {
                 return Sequence.prototype.displayTabTooltip.apply(self, [event]);
             };
+            this.arrowKeys = {
```

Comment:
Could this use HtmlUtils.constants? http://ui-toolkit.edx.org/utilities/constants/